resonance/nonlinear_systems.py

```diff
@@ -0,0 +1,42 @@
+from inspect import getargspec
+
+import numpy as np
+import scipy as sp
+import scipy.integrate  # scipy doesn't import automatically
+
+from .systems import _System
+
+
+class NonLinearSystem(_System):
+
+    def _integrate_equations_of_motion(self, times, integrator='odeint'):
+
+        x0 = list(self.coordinates.values())[0]
+        v0 = list(self.speeds.values())[0]
+
+        initial_conditions = np.array([x0, v0])
```

Comment:
We need some way to enforce or detect the order of the state equations. Will be especially important for MDoF systems.